docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -246,7 +246,7 @@ Connection plugins allow Ansible to connect to the target hosts so it can execut
 
 Ansible version 2.1 introduced the ``smart`` connection plugin. The ``smart`` connection type allows Ansible to automatically select either the ``paramiko`` or ``openssh`` connection plugin based on system capabilities, or the ``ssh`` connection plugin if OpenSSH supports ControlPersist.
 
-To create a new connection plugin (for example, to support SNMP, Message bus, or other transports), copy the format of one of the existing connection plugins and drop it into the ``connection_plugins`` directory.
+To create a new connection plugin (for example, to support SNMP, Message bus, or other transports), copy the format of one of the existing connection plugins and drop it into the ``plugins/connection`` directory.
```

Comment:
This change is not necessarily true.  By default we will utilize a `connection_plugins` directory relative to the playbook.  `plugins/connection` has no real meaning. The default connection plugin path is defined as `~/.ansible/plugins/connection:/usr/share/ansible/plugins/connection`

Reply:
Hrm, I suppose there is more ambiguity here.  As this doesn't indicate the purpose of creating the connection plugin.  Whether it is to contribute back to Ansible, or just for personal development.

Reply:
When I read `connection_plugins` I read it as the path to the source (`/lib/ansible/plugins/connection/`) and not where the plugins are kept after installation. If I'm developing a plugin, I am thinking in the context of the source tree and not the installation path.